hal/src/samd51/mod.rs

```diff
@@ -4,8 +4,12 @@ pub mod pwm;
 pub mod sercom;
 pub mod timer;
 pub mod trng;
+pub mod watchdog;
 
-#[cfg(feature="unproven")]
+mod serial_number;
```

Comment:
Why are you doing this instead of just putting `pub mod serial_number`?

Reply:
I was under the impression that `pub mod serial_number` would mean consumers would need to do `hal::serial_number::serial_number()` instead of `hal::serial_number()`, which stutters?

Reply:
oh yeah I guess